i18n/en.json

```diff
@@ -18,13 +18,13 @@
 	"wiki-seo-param-title_separator": "Title separator:",
 	"wiki-seo-param-title_separator-description": "[https://www.mediawiki.org/wiki/Extension:WikiSEO#Title_Modes More Information]",
 	"wiki-seo-param-keywords": "Keywords:",
-	"wiki-seo-param-keywords-description": "This attribute controls the content of the <code>&lt;meta type='keywords'&gt;</code> element.<br>The content is further used for the <code>article:tag</code> element.",
+	"wiki-seo-param-keywords-description": "This attribute controls the content of the <code>keywords</code> and <code>article:tag</code> elements.",
 	"wiki-seo-param-description": "Article description:",
-	"wiki-seo-param-description-description": "This attribute controls the content of the <code>description</code> element.<br>It is further used for the <code>og:description</code> and <code>twitter:description</code> elements.",
+	"wiki-seo-param-description-description": "This attribute controls the content of the <code>description</code> and <code>og:description</code> elements.",
 	"wiki-seo-param-robots": "Crawler mode (robots):",
 	"wiki-seo-param-google_bot": "GoogleBot modue:",
 	"wiki-seo-param-image": "Article image:",
-	"wiki-seo-param-image-description": "This attribute controls the content of the <code>og:image</code> and <code>twitter:image</code> element.<br>This image is mostly displayed as a thumbnail on social media.",
```

Comment:
Can you remove those keys also from `qqq.json`?

`qqq.json` is used by translate wiki as a starting point for translations. If the keys are present there, they will appear in other translations sooner or later